### ISSUE TITLE
Ideas why results are unsatisfactory?

### ISSUE THREAD

**Issue by blutjens** [yzhq97/cnn-registration#24]
Hello, 
Thanks for the model! I ran it to align two "satellite" images from different sensors (NAIP, DigitalGlobe). Unfortunately, the results look pretty unsatisfactory (see image). Do you have any ideas/thoughts how the results could be increased?

![cnn_registration_w_hist](https://user-images.githubusercontent.com/29470205/92670232-92a3e780-f2e1-11ea-8d9e-ee692cfde901.png)

Thank you!
Björn

**Comment by ormosia6** [comment]
Have you solved the problem?

**Comment by blutjens** [comment]
No. My assumption is that the method would need to be retrained on my dataset. I ended up matching the imagery via standard methods in opencv.
